katas/content/complex_arithmetic/complex_exponents/Placeholder.qs

```diff
@@ -1,8 +1,8 @@
 namespace Kata {
     open Microsoft.Quantum.Math;
-    
+
     function ComplexExponent(x : Complex) : Complex {
         // Implement your solution here...
-        return Complex(0.0, 0.0);
+        return Complex(0., 0.);
```

Comment:
Is changing `0.0` to `0.` really needed? I'm curious why the preference for this style. I tend to lean the other way, personally.

Reply:
Oh, never mind. I see other katas already do this, so this change is making it consistent.